mm2src/database/stats_swaps.rs

```diff
@@ -1,13 +1,18 @@
-use crate::mm2::lp_swap::{MakerSavedSwap, SavedSwap, SavedSwapIo, TakerSavedSwap};
+use crate::mm2::{database::SqlResult,
+                 lp_swap::{MakerSavedSwap, SavedSwap, SavedSwapIo, TakerSavedSwap}};
+
 use common::{log::{debug, error},
-             mm_ctx::MmArc};
+             mm_ctx::MmArc,
+             mm_number::BigDecimal};
 use db_common::sqlite::rusqlite::{Connection, OptionalExtension};
 use std::collections::HashSet;
 
 const CREATE_STATS_SWAPS_TABLE: &str = "CREATE TABLE IF NOT EXISTS stats_swaps (
     id INTEGER NOT NULL PRIMARY KEY,
     maker_coin VARCHAR(255) NOT NULL,
     taker_coin VARCHAR(255) NOT NULL,
+    maker_coin_usd_price DECIMAL,
+    taker_coin_usd_price DECIMAL,
```

Comment:
I'm totally unsure, but these fields should be arranged somehow differently 😅 
Maybe like this
```
    id INTEGER NOT NULL PRIMARY KEY,
    maker_coin VARCHAR(255) NOT NULL,
    maker_coin_usd_price DECIMAL,
    taker_coin VARCHAR(255) NOT NULL,
    taker_coin_usd_price DECIMAL,
```
Or they need to be placed in the end
```
    maker_amount DECIMAL NOT NULL,
    taker_amount DECIMAL NOT NULL,
    is_success INTEGER NOT NULL,
    maker_coin_usd_price DECIMAL,
    taker_coin_usd_price DECIMAL
```
Please consider moving them :)